test/data/data.ts

```diff
@@ -104,29 +104,29 @@ export const h: ResponseToolkit = {
  * Mock Consent Resources
  */
 export const partialConsentActive: Consent = {
-  id: '1234',
+  id: 'b51ec534-ee48-4575-b6a9-ead2955b8069',
   initiatorId: 'pisp-2342-2233',
   participantId: 'dfsp-3333-2123',
   status: 'ACTIVE'
 }
 
 export const partialConsentActiveConflictingInitiatorId: Consent = {
-  id: '1234',
+  id: 'b51ec534-ee48-4575-b6a9-ead2955b8069',
   initiatorId: 'pi2-2233',
   participantId: 'dfs333-2123',
   status: 'ACTIVE'
 }
 
 export const partialConsentRevoked: Consent = {
-  id: '1234',
+  id: 'b51ec534-ee48-4575-b6a9-ead2955b8069',
   initiatorId: 'pisp-2342-2233',
   participantId: 'dfsp-3333-2123',
   revokedAt: '2020-08-19T05:44:18.843Z',
   status: 'REVOKED'
 }
 
 export const completeConsentRevoked: Consent = {
-  id: '1234',
+  id: 'b51ec534-ee48-4575-b6a9-ead2955b8069',
   initiatorId: 'pisp-2342-2233',
   participantId: 'dfsp-3333-2123',
   status: 'REVOKED',
```

Comment:
@lewisdaly if we drop `revoke` endpoint shouldn't we also update the `Consent.status` enumeration -> if we use DELETE method, maybe status should be DELETED not REVOKED. ?